src/main/webapp/cdn/blockly/generators/propc/gpio.js

```diff
@@ -991,7 +991,7 @@ Blockly.propc.ab_drive_init = function () {
 Blockly.Blocks.ab_drive_ramping = {
     helpUrl: Blockly.MSG_ROBOT_HELPURL,
     init: function () {
-        this.setTooltip(Blockly.MSG_ROBOT_DRIVE_INIT_TOOLTIP);
```

Comment:
Are the deleted tool tip message being used elsewhere? I do not see a corresponding removal of the deleted tool tips in the language file.